### PR TITLE
Many to many relationship example

### PR DESCRIPTION
**Pull request by rwpswami**

### CODE REVIEW

File: src/main/java/com/pracore/user/controllers/UserController.java [hunk 1]
```diff
@@ -18,21 +16,12 @@
...
         return users;
     }
 
-    @GetMapping("/hobbies")
-    public List<Hobby> hobbies() {
-        return hobbyRepository.findAll();
-    }
-
     @PostMapping("/users")
     public ResponseEntity<User> createUser(@RequestBody User user, BindingResult result) {
```
Comment (mohitlaad-robustwealth) [inline src/main/java/com/pracore/user/controllers/UserController.java:26]: what is the use of BindingResult>

File: src/main/java/com/pracore/user/services/HobbyService.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class HobbyService {
+
+    @Autowired
+    HobbyRepository hobbyRepository;
```
Comment (mohitlaad-robustwealth) [inline src/main/java/com/pracore/user/services/HobbyService.java:14]: we can make it private.

File: src/main/java/com/pracore/user/controllers/UserController.java (hunk 1 above)
Comment (rwpswami) [reply src/main/java/com/pracore/user/controllers/UserController.java:26]: I read somewhere. If passed request payload doesn't match to given model. In this case if it doesn't match to User model. Then we have `result.hasErrors` method to catch this

Comment (rachit-rw) [reply src/main/java/com/pracore/user/controllers/UserController.java:26]: that is more for UI forms, if you create spring mvc. Not needed for microservice.

**Review by anandp47 — APPROVED** [review]

**Pull request merged**